README.md

```diff
@@ -20,35 +20,44 @@ DataLens requires Docker to be installed. Follow these instructions depending on
 
 ### Running containers
 
-Use the following command to start DataLens containers:
+Clone repository:
 
 ```bash
 git clone https://github.com/datalens-tech/datalens && cd datalens
+```
+
+Then, for a quick start, use the following command to start DataLens containers:
```

Comment:
May be this would be a little bit better?
```
For the quick start use the following command to start DataLens containers:
```
at least you don't need commas ;)